tests_v2/test_tmux_compat_geometry.py

```
@@ -0,0 +1,247 @@
+
+    # Each pane should be roughly half the original width
+    cols1 = p1["columns"]
+    cols2 = p2["columns"]
+    _must(cols1 > 0 and cols2 > 0, f"Columns should be > 0, got {cols1} and {cols2}")
+    _must(cols1 < single_cols, f"Split pane cols ({cols1}) should be less than original ({single_cols})")
+
+    # Verify tmux-compat format also shows two lines with different pane_left
+    fmt = "#{pane_id}\t#{pane_width}\t#{pane_left}"
+    proc = _run_tmux_compat(cli, ["list-panes", "-t", f"%{p1['id']}", "-F", fmt])
+    _must(proc.returncode == 0, f"list-panes after split failed: {proc.stderr}")
+    lines = [l for l in proc.stdout.strip().split("\n") if l.strip()]
```

Ruff (0.15.7)

[error] Ambiguous variable name: `l`

(E741)